addons/traefik/1.7.x/traefik-14.yaml

```diff
@@ -61,6 +61,8 @@ spec:
           cpu: 1000m
         requests:
           cpu: 500m
+      accessLogs:
+        enabled: false
```

Comment:
doesn't this **disable** access logs?

Reply:
Oh you are right, I've fixed it and changed the value to `true`.